`node android` fails due to wrong gradle version?

- [x] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and did not find a similar issue.

**Bug in mobile app**
Related to mobile app, but to the build process.

**Describe the bug**
Build fails.

**To Reproduce**
<details>
<summary>Show full log</summary>

```sh
@Osiris-Team ➜ /workspaces/tutanota (master) $ npm ci
npm WARN skipping integrity check for git dependency ssh://git@github.com/tutao/better-sqlite3-sqlcipher.git 
npm WARN deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm WARN deprecated domexception@4.0.0: Use your platform's native DOMException instead

> tutanota@218.240227.0 preinstall
> git config core.hooksPath githooks || true


> tutanota@218.240227.0 postinstall
> node buildSrc/postinstall.js

npm list --json > node_modules/.npm-deps-resolved

added 930 packages, and audited 938 packages in 57s

113 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice 
npm notice New minor version of npm available! 10.2.4 -> 10.4.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.4.0
npm notice Run npm install -g npm@10.4.0 to update!
npm notice 
@Osiris-Team ➜ /workspaces/tutanota (master) $ npm run build-packages

> tutanota@218.240227.0 build-packages
> node buildSrc/buildPackages.js all

$ npx tsc -b ./packages/licc ./packages/otest ./packages/tutanota-crypto ./packages/tutanota-error ./packages/tutanota-test-utils ./packages/tutanota-usagetests ./packages/tutanota-utils
@Osiris-Team ➜ /workspaces/tutanota (master) $ node android

BUILD: Starting prod build with build type: release, webclient: dist, host: undefined 

started cleaning 0
bundling polyfill 0.002
(typescript plugin) @rollup/plugin-typescript: outputToFilesystem option is defaulting to true.
started copying images 23.827
started bundling 23.88
Circular dependency: src/gui/base/Dialog.ts -> src/gui/base/TextField.ts -> src/gui/base/GuiUtils.ts -> src/gui/base/Dialog.ts
Circular dependency: src/gui/base/TextField.ts -> src/gui/base/GuiUtils.ts -> src/gui/base/DropDownSelector.ts -> src/gui/base/TextField.ts
Circular dependency: src/gui/base/GuiUtils.ts -> src/gui/base/DropDownSelector.ts -> src/gui/base/GuiUtils.ts
Circular dependency: src/login/LoginView.ts -> src/login/LoginViewModel.ts -> src/login/LoginView.ts
Circular dependency: src/calendar/gui/eventeditor-model/CalendarNotificationModel.ts -> src/calendar/gui/eventeditor-model/CalendarEventModel.ts -> src/calendar/gui/eventeditor-model/CalendarNotificationModel.ts
Circular dependency: src/calendar/gui/eventeditor-model/CalendarEventModel.ts -> src/calendar/gui/eventeditor-model/CalendarEventModelStrategy.ts -> src/calendar/gui/eventeditor-model/CalendarEventModel.ts
bundling timings: 
# BUILD 17647.32096
## initialize 3705.126134000002
- plugin 1 (typescript) - buildStart 3704.7526979999966
- plugin 3 (commonjs) - buildStart 0.070982000001095
- plugin 7 (node-resolve) - buildStart 0.032461000002513174
## generate module graph 8949.512210999997
- plugin 0 (commonjs--resolver) - resolveId 208.42502599971704
- plugin 1 (typescript) - resolveId 504.4378899998519
- plugin 2 (resolve-libs) - resolveId 28.735193999858893
- plugin 7 (node-resolve) - resolveId 342.5075660000475
- plugin 1 (typescript) - load 55.084787000021606
- plugin 3 (commonjs) - transform 297.3043900001394
generate ast 18517.938320000165
analyze ast 2315.3406300001006
- plugin 3 (commonjs) - load 1.8707420000318962
- plugin 7 (node-resolve) - load 1.2042160000601143
## sort and bind modules 286.67059799999697
## mark included statements 4675.224462999999
treeshaking pass 1 1260.6799919999976
treeshaking pass 2 1024.9987019999971
treeshaking pass 3 428.4516829999993
treeshaking pass 4 500.55036200000177
treeshaking pass 5 228.09107900000527
treeshaking pass 6 161.07561700000224
treeshaking pass 7 147.96161900000152
treeshaking pass 8 136.35842199999752
treeshaking pass 9 148.61557500000345
treeshaking pass 10 180.18344400000205
treeshaking pass 11 131.49711799999932
treeshaking pass 12 198.0735380000042
treeshaking pass 13 127.92326700000558
- plugin 1 (typescript) - buildEnd 19.305365000000165
- plugin 3 (commonjs) - buildEnd 0.029986000001372304
started writing bundles 41.594
app.js  90.6015625K
        /src/misc/ClientDetector.ts
        libs/mithril.js?commonjs-module
        libs/mithril.js
        /src/misc/WhitelabelCustomizations.ts
        /src/misc/LanguageViewModel.ts
        /src/gui/animation/Easing.ts
        /src/gui/base/Color.ts
        /src/gui/animation/Animations.ts
        /src/misc/DeviceConfig.ts
        /src/gui/base/Logo.ts
        /src/gui/builtinThemes.ts
        /src/gui/ThemeController.ts
        /src/gui/theme.ts
        /src/misc/KeyManager.ts
        /src/misc/WindowFacade.ts
        /src/gui/base/Modal.ts
        /src/gui/size.ts
        /src/misc/Log.ts
        /src/gui/styles.ts
        /src/gui/HtmlUtils.ts
        /src/gui/base/Overlay.ts
        /src/RootView.ts
        /src/misc/ErrorHandler.ts
        /src/ApplicationPaths.ts
        /src/app.ts
        /src/api/main/LoginController.ts
        /src/misc/RouteChange.ts
        /src/misc/ClientConstants.ts
        /src/gui/mixins.ts
        /src/gui/AriaUtils.ts
worker.js  198.921875K
        /src/api/worker/EventQueue.ts
        /src/api/worker/ProgressMonitorDelegate.ts
        /src/api/worker/EventBusClient.ts
        /src/api/worker/rest/DefaultEntityRestCache.ts
        /src/api/worker/rest/EntityRestClient.ts
        /packages/tutanota-crypto/dist/internal/sjcl.js
        /packages/tutanota-crypto/dist/random/Randomizer.js
        /packages/tutanota-crypto/dist/hashes/Sha256.js
        /packages/tutanota-crypto/dist/misc/Utils.js
        /packages/tutanota-crypto/dist/hashes/Sha512.js
        /packages/tutanota-crypto/dist/encryption/Aes.js
        /packages/tutanota-crypto/dist/internal/noble-curves-1.3.0.js
        /packages/tutanota-crypto/dist/encryption/Ecc.js
        /packages/tutanota-crypto/dist/internal/bCrypt.js
        /packages/tutanota-crypto/dist/misc/Constants.js
        /packages/tutanota-crypto/dist/hashes/Bcrypt.js
        /packages/tutanota-crypto/dist/encryption/Liboqs/Kyber.js
        /packages/tutanota-crypto/dist/encryption/Liboqs/KyberKeyPair.js
        /packages/tutanota-crypto/dist/hashes/Argon2id/Argon2id.js
        /packages/tutanota-crypto/dist/random/SecureRandom.js
        /packages/tutanota-crypto/dist/internal/crypto-jsbn-2012-08-09_1.js
        /packages/tutanota-crypto/dist/encryption/AsymmetricKeyPair.js
        /packages/tutanota-crypto/dist/encryption/Rsa.js
        /packages/tutanota-crypto/dist/encryption/KeyEncryption.js
        /packages/tutanota-crypto/dist/encryption/PQKeyPairs.js
        /packages/tutanota-crypto/dist/hashes/Sha1.js
        /packages/tutanota-crypto/dist/misc/TotpVerifier.js
        /packages/tutanota-crypto/dist/hashes/MurmurHash.js
        /packages/tutanota-crypto/dist/hashes/HKDF.js
        /packages/tutanota-crypto/dist/index.js
        /src/api/worker/facades/PQMessage.ts
        /src/api/worker/crypto/CryptoFacade.ts
        /src/api/worker/facades/LoginFacade.ts
        /src/api/worker/rest/RestClient.ts
        /src/api/worker/SuspensionHandler.ts
        /src/api/worker/facades/DeviceEncryptionFacade.ts
        /src/native/worker/AesApp.ts
        /src/api/worker/crypto/RsaImplementation.ts
        /src/api/worker/Compression.ts
        /src/api/worker/crypto/InstanceMapper.ts
        /src/api/worker/rest/AdminClientDummyEntityRestCache.ts
        /src/api/worker/utils/SleepDetector.ts
        /src/api/worker/rest/CustomCacheHandler.ts
        /src/api/worker/rest/EphemeralCacheStorage.ts
        /src/api/worker/rest/CacheStorageProxy.ts
        /src/api/worker/rest/ServiceExecutor.ts
        /src/api/worker/facades/UserFacade.ts
        /src/api/worker/offline/SqlValue.ts
        /src/api/worker/offline/OfflineStorage.ts
        /src/api/worker/offline/StandardMigrations.ts
        /src/api/worker/offline/migrations/sys-v94.ts
        /src/api/worker/offline/migrations/tutanota-v66.ts
        /src/api/worker/offline/migrations/sys-v92.ts
        /src/api/worker/offline/migrations/tutanota-v65.ts
        /src/api/worker/offline/migrations/sys-v91.ts
        /src/api/worker/offline/migrations/sys-v90.ts
        /src/api/worker/offline/migrations/tutanota-v64.ts
        /src/api/worker/offline/OfflineStorageMigrator.ts
        /src/api/worker/facades/EntropyFacade.ts
        /src/api/worker/facades/BlobAccessTokenFacade.ts
        /src/api/worker/crypto/OwnerEncSessionKeysUpdateQueue.ts
        /src/api/worker/EventBusEventCoordinator.ts
        /src/api/worker/facades/WorkerFacade.ts
        /src/api/worker/facades/Argon2idFacade.ts
        /src/api/worker/facades/KyberFacade.ts
        /src/api/worker/facades/PQFacade.ts
        /src/api/worker/WorkerLocator.ts
        /src/api/worker/WorkerImpl.ts
        /src/api/worker/worker.ts
        /src/native/worker/RsaApp.ts
        /src/api/worker/DateProvider.ts
        /src/api/worker/utils/DbUtils.ts
calendar-view-vM0-Ai4P.js  56.6923828125K
        /src/calendar/view/CalendarRow.ts
        /src/calendar/view/EventDetailsView.ts
        /src/calendar/view/CalendarEventBubble.ts
        /src/calendar/view/ContinuingCalendarEventBubble.ts
        /src/calendar/view/EventDragHandler.ts
        /src/calendar/view/CalendarMonthView.ts
        /src/calendar/view/CalendarAgendaItemView.ts
        /src/calendar/view/CalendarTimeIndicator.ts
        /src/calendar/view/CalendarAgendaView.ts
        /src/calendar/view/CalendarDayEventsView.ts
        /src/calendar/view/MultiDayCalendarView.ts
        /src/calendar/view/TodayIconButton.ts
        /src/calendar/view/CalendarMobileHeader.ts
        /src/calendar/view/CalendarDesktopToolbar.ts
        /src/calendar/view/CalendarView.ts
        /src/calendar/view/CalendarNotificationSender.ts
        /src/calendar/view/CalendarViewModel.ts
        /src/calendar/view/CalendarInvites.ts
common-min-g4mXDxm1.js  38.853515625K
        /packages/tutanota-error/dist/TutanotaError.js
        /packages/tutanota-error/dist/index.js
        /src/api/common/error/ProgrammingError.ts
        /src/api/common/Env.ts
        /packages/tutanota-utils/dist/TypeRef.js
        /packages/tutanota-utils/dist/Utils.js
        /packages/tutanota-utils/dist/MapUtils.js
        /packages/tutanota-utils/dist/ArrayUtils.js
        /packages/tutanota-utils/dist/AsyncResult.js
        /packages/tutanota-utils/dist/CollectionUtils.js
        /packages/tutanota-utils/dist/DateUtils.js
        /packages/tutanota-utils/dist/Encoding.js
        /packages/tutanota-utils/dist/LazyLoaded.js
        /packages/tutanota-utils/dist/PromiseMap.js
        /packages/tutanota-utils/dist/PromiseUtils.js
        /packages/tutanota-utils/dist/SortedArray.js
        /packages/tutanota-utils/dist/StringUtils.js
        /packages/tutanota-utils/dist/WebAssembly.js
        /packages/tutanota-utils/dist/MathUtils.js
        /packages/tutanota-utils/dist/Csv.js
        /packages/tutanota-utils/dist/Tokenizer.js
        /packages/tutanota-utils/dist/index.js
        libs/stream.js?commonjs-module
        libs/stream.js
        /src/api/common/TutanotaConstants.ts
        /src/api/common/Logger.ts
        /src/api/common/error/RestError.ts
        /src/api/common/error/SuspensionError.ts
        /src/api/common/error/LoginIncompleteError.ts
        /src/api/common/error/SessionKeyNotFoundError.ts
        /src/api/common/error/SseError.ts
        /src/api/common/error/RecipientsNotFoundError.ts
        /src/api/common/error/RecipientNotResolvedError.ts
        /src/api/common/error/OfflineDbClosedError.ts
        /src/api/common/error/OutOfSyncError.ts
        /src/api/common/error/DbError.ts
        /src/api/common/error/IndexingNotSupportedError.ts
        /src/api/common/error/QuotaExceededError.ts
        /src/api/common/error/CancelledError.ts
        /src/api/common/error/FileOpenError.ts
        /src/api/common/error/DeviceStorageUnavailableError.ts
        /src/api/common/error/MailBodyTooLargeError.ts
        /src/api/common/error/ImportError.ts
        /src/api/common/error/WebauthnError.ts
        /src/api/common/error/PermissionError.ts
        /src/api/common/error/FileNotFoundError.ts
        /src/api/common/error/CredentialAuthenticationError.ts
        /src/api/common/error/KeyPermanentlyInvalidatedError.ts
        /src/misc/parsing/ParserCombinator.ts
        /src/api/common/error/SetupMultipleError.ts
        /src/api/common/error/ParsingError.ts
        /src/api/common/error/InvalidDatabaseStateError.ts
        /src/api/common/error/MembershipRemovedError.ts
contacts-t9t14rkB.js  61.6455078125K
        /src/contacts/view/ContactGuiUtils.ts
        /src/contacts/ContactAggregateEditor.ts
        /src/contacts/ContactEditor.ts
        /src/contacts/view/ContactRow.ts
        /src/contacts/view/ContactListView.ts
        /src/contacts/view/MultiContactViewer.ts
        /src/contacts/view/ContactMergeView.ts
        /src/contacts/ContactMergeUtils.ts
        /src/contacts/VCardExporter.ts
        /src/contacts/view/ContactViewer.ts
        /src/contacts/view/ContactCardViewer.ts
        /src/contacts/view/MobileActionBar.ts
        /src/contacts/view/ContactViewerActions.ts
        /src/contacts/VCardImporter.ts
        /src/contacts/view/ImportAsVCard.ts
        /src/contacts/view/ContactListRecipientView.ts
        /src/contacts/ContactListEditor.ts
        /src/contacts/view/ContactListEntryViewer.ts
        /src/contacts/view/ContactView.ts
        /src/contacts/view/ContactViewModel.ts
        /src/contacts/view/ContactListViewModel.ts
common-4v31ka00.js  365.970703125K
        /packages/tutanota-crypto/dist/misc/CryptoError.js
        /packages/tutanota-crypto/dist/error.js
        /src/api/common/utils/ErrorUtils.ts
        /src/api/common/threading/MessageDispatcher.ts
        /src/api/common/EntityConstants.js
        /src/api/common/utils/EntityUtils.ts
        /src/api/common/EntityFunctions.ts
        /src/api/common/utils/EntityUpdateUtils.ts
        /src/api/common/utils/ProgressMonitor.ts
        /src/api/common/EntityClient.ts
        /src/api/common/utils/BirthdayUtils.ts
        /src/api/common/utils/Scheduler.ts
        /src/api/common/utils/NoZoneDateProvider.ts
        libs/cborg.js
        /src/api/common/mail/FolderSystem.ts
        /src/api/common/MailWrapper.ts
        /src/api/common/mail/CommonMailUtils.ts
        /src/api/common/DomainConfigProvider.ts
        /src/api/common/WorkerProxy.ts
        /src/api/common/threading/Transport.ts
        /src/api/common/utils/GroupUtils.ts
        /src/api/common/utils/CommonCalendarUtils.ts
        /src/misc/credentials/DatabaseKeyFactory.ts
        /src/api/common/CountryList.ts
        /src/api/common/DataFile.ts
        /src/api/common/utils/FileUtils.ts
        /src/api/common/utils/CustomerUtils.ts
        /src/offline/CachePostLoginAction.ts
        /src/api/common/utils/PlainTextSearch.ts
        /src/api/common/threading/CrossOriginTransport.ts
        /src/desktop/config/ConfigKeys.ts
        /src/api/common/utils/CommonFormatter.ts
        /src/api/common/ExposeNativeInterface.ts
        /src/offline/LoadingState.ts
        /src/api/common/utils/ObservableLazyLoaded.ts
date-brtlBV80.js  82.1943359375K
        libs/luxon.js
        /src/calendar/date/Time.ts
        /src/calendar/date/CalendarUtils.ts
        /src/calendar/date/CalendarEventsRepository.ts
        /src/calendar/date/AlarmScheduler.ts
gui-base-_jExEZnA.js  163.69921875K
        /src/gui/base/icons/Icons.ts
        /src/gui/base/buttons/BaseButton.ts
        /src/gui/base/Button.ts
        /src/gui/base/DialogHeaderBar.ts
        /src/gui/base/PureComponent.ts
        /src/gui/base/icons/BootIcons.ts
        /src/gui/base/Icon.ts
        /src/gui/base/Dropdown.ts
        /src/gui/base/IconButton.ts
        /src/gui/base/DropDownSelector.ts
        /src/gui/base/GuiUtils.ts
        /src/gui/base/TextField.ts
        /src/gui/base/DialogInjectionRight.ts
        /src/gui/base/Dialog.ts
        /src/gui/base/buttons/LoginButton.ts
        /src/gui/base/ExternalLink.ts
        /src/gui/base/NotificationOverlay.ts
        /src/gui/base/ProgressBar.ts
        /src/gui/base/OfflineIndicatorViewModel.ts
        /src/gui/base/SnackBar.ts
        /src/gui/base/Checkbox.ts
        /src/gui/base/Expander.ts
        /src/gui/base/buttons/BubbleButton.ts
        /src/gui/base/icons/FontIcons.ts
        /src/gui/base/ViewColumn.ts
        /src/gui/base/NavBar.ts
        /src/gui/base/NavButton.ts
        /src/gui/base/OfflineIndicator.ts
        /src/gui/base/DesktopBaseHeader.ts
        /src/gui/base/buttons/ToggleButton.ts
        /src/gui/base/SwipeHandler.ts
        /src/gui/base/ListUtils.ts
        /src/gui/base/ListSwipeHandler.ts
        /src/gui/base/List.ts
        /src/gui/base/ColumnEmptyMessageBox.ts
        /src/gui/base/RichTextToolbar.ts
        /src/gui/base/TextDisplayArea.ts
        /src/gui/base/CounterBadge.ts
        /src/gui/base/BubbleTextField.ts
        /src/gui/base/StatusField.ts
        /src/gui/base/InfoBanner.ts
        /src/gui/base/buttons/ArrowButton.ts
        /src/gui/base/MessageBox.ts
        /src/gui/base/Badge.ts
        /src/gui/base/RecipientButton.ts
        /src/gui/base/Table.ts
        /src/gui/base/RadioSelector.ts
        /src/gui/base/WizardDialog.ts
        /src/gui/base/SegmentControl.ts
        /src/gui/base/BaseSearchBar.ts
        /src/gui/base/ColorPicker.ts
        /src/gui/base/ActionBar.ts
        /src/gui/base/buttons/BannerButton.ts
        /src/gui/base/PageView.ts
        /src/gui/base/CalendarSwipeHandler.ts
        /src/gui/base/Carousel.ts
        /src/gui/base/IconSegmentControl.ts
        /src/gui/base/NotFoundPage.ts
invoice-wYjpwPd7.js  26.9462890625K
        /src/api/worker/pdf/PdfConstants.ts
        /src/api/worker/pdf/PdfObject.ts
        /src/api/worker/pdf/PdfStreamObject.ts
        /src/api/worker/pdf/Deflater.ts
        /src/api/worker/pdf/PdfWriter.ts
        /src/api/worker/pdf/PdfDocument.ts
        /src/api/worker/invoicegen/InvoiceTexts.ts
        /src/api/worker/invoicegen/PdfInvoiceGenerator.ts
date-gui-Gzrcvsyx.js  110.53125K
        /src/misc/DateParser.ts
        /src/calendar/gui/CalendarGuiUtils.ts
        /src/calendar/gui/pickers/DatePicker.ts
        /src/calendar/gui/pickers/DatePickerDialog.ts
        /src/calendar/gui/eventpopup/EventPreviewView.ts
        /src/calendar/gui/eventeditor-model/CalendarEventWhenModel.ts
        /src/calendar/gui/eventeditor-model/CalendarEventWhoModel.ts
        /src/calendar/gui/eventeditor-model/CalendarEventAlarmModel.ts
        /src/calendar/gui/eventeditor-model/CalendarEventModelStrategy.ts
        /src/calendar/gui/eventeditor-model/CalendarEventModel.ts
        /src/calendar/gui/eventeditor-model/CalendarNotificationModel.ts
        /src/calendar/gui/eventeditor-view/AttendeeListEditor.ts
        /src/calendar/gui/pickers/TimePicker.ts
        /src/calendar/gui/eventeditor-view/EventTimeEditor.ts
        /src/calendar/gui/eventeditor-view/RepeatRuleEditor.ts
        /src/calendar/gui/eventeditor-view/CalendarEventEditView.ts
        /src/calendar/gui/eventeditor-view/CalendarEventEditDialog.ts
        /src/calendar/gui/day-selector/DaySelector.ts
        /src/calendar/gui/EditCalendarDialog.ts
        /src/calendar/export/WindowsZones.ts
        /src/calendar/export/CalendarParser.ts
        /src/calendar/export/CalendarExporter.ts
        /src/calendar/export/CalendarImporter.ts
        /src/calendar/export/CalendarImporterDialog.ts
        /src/calendar/gui/eventpopup/CalendarEventPopup.ts
        /src/calendar/gui/day-selector/DaySelectorSidebar.ts
        /src/calendar/gui/day-selector/DaySelectorPopup.ts
        /src/calendar/model/CalendarModel.ts
        /src/calendar/gui/eventpopup/CalendarEventPreviewViewModel.ts
linkify-h7icspiz.js  33.240234375K
        libs/linkify.js
        libs/linkify-html.js
        /src/api/worker/Urlifier.ts
login-L0dBcvPO.js  35.701171875K
        /src/login/PostLoginUtils.ts
        /src/login/PostLoginActions.ts
        /src/login/LoginViewModel.ts
        /src/login/LoginForm.ts
        /src/login/CredentialsSelector.ts
        /src/login/LoginLogDialog.ts
        /src/login/MigratingCredentialsBanner.ts
        /src/login/LoginView.ts
        /src/login/ExternalLoginView.ts
        /src/login/NativeWebauthnView.ts
        /src/login/MobileWebauthnView.ts
        /src/login/CredentialsMigrationViewModel.ts
        /src/login/CredentialsMigrationView.ts
        /src/login/CredentialRemovalHandler.ts
mail-editor-h_g6drwz.js  119.73046875K
        libs/squire-raw.mjs
        /src/gui/editor/Editor.ts
        /src/gui/editor/HtmlEditor.ts
        /src/mail/signature/Signature.ts
        /src/mail/editor/MailEditorViewModel.ts
        /src/templates/model/TemplatePopupModel.ts
        /src/templates/view/TemplateConstants.ts
        /src/templates/view/TemplatePopupResultRow.ts
        /src/templates/view/TemplateExpander.ts
        /src/templates/view/TemplateSearchBar.ts
        /src/templates/TemplateGroupUtils.ts
        /src/templates/view/TemplatePopup.ts
        /src/templates/view/TemplateShortcutListener.ts
        /src/knowledgebase/view/KnowledgeBaseListEntry.ts
        /src/knowledgebase/view/KnowledgeBaseEntryView.ts
        /src/knowledgebase/view/KnowledgeBaseDialogContent.ts
        /src/knowledgebase/view/KnowledgeBaseDialog.ts
        /src/knowledgebase/model/KnowledgeBaseSearchFilter.ts
        /src/knowledgebase/model/KnowledgeBaseModel.ts
        /src/mail/editor/MailEditor.ts
        /src/mail/editor/SendMailModel.ts
        /src/mail/press/PressReleaseEditor.ts
jszip-TGEAmg11.js  94.8203125K
        libs/jszip.js?commonjs-module
        libs/jszip.js
mail-view-PGDZ55zi.js  106.8310546875K
        /src/mail/view/MailReportDialog.ts
        /src/mail/view/MailGuiUtils.ts
        /src/mail/view/MinimizedEditorOverlay.ts
        /src/mail/view/MinimizedMailEditorOverlay.ts
        /src/mail/view/MailRow.ts
        /src/mail/export/Bundler.ts
        /src/mail/export/Exporter.ts
        /src/mail/view/MailListView.ts
        /src/mail/view/MultiItemViewer.ts
        /src/mail/view/MailViewerUtils.ts
        /src/mail/view/MobileMailActionBar.ts
        /src/mail/view/EditFolderDialog.ts
        /src/mail/view/MailFolderRow.ts
        /src/mail/view/MailFoldersView.ts
        /src/mail/view/EditFoldersDialog.ts
        /src/mail/view/EventBanner.ts
        /src/mail/view/MailViewerHeader.ts
        /src/mail/view/MailViewer.ts
        /src/mail/view/CollapsedMailView.ts
        /src/mail/view/ConversationViewer.ts
        /src/mail/view/MailViewerToolbar.ts
        /src/mail/view/MobileMailMultiselectionActionBar.ts
        /src/mail/view/MailFilterButton.ts
        /src/mail/view/MailView.ts
        /src/mail/view/MailViewerViewModel.ts
        /src/mail/view/MailViewModel.ts
        /src/mail/view/ConversationViewModel.ts
native-common-Pv5M4cD9.js  16.548828125K
        /src/native/common/FileApp.ts
        /src/native/common/generatedipc/NativeCryptoFacadeSendDispatcher.ts
        /src/native/common/generatedipc/FileFacadeSendDispatcher.ts
        /src/native/common/generatedipc/NativePushFacadeSendDispatcher.ts
        /src/native/common/generatedipc/ExportFacadeSendDispatcher.ts
        /src/native/common/generatedipc/InterWindowEventFacadeSendDispatcher.ts
        /src/native/common/generatedipc/SqlCipherFacadeSendDispatcher.ts
        /src/native/common/generatedipc/NativeCredentialsFacadeSendDispatcher.ts
        /src/native/common/generatedipc/WebAuthnFacadeSendDispatcher.ts
        /src/native/common/NativeLineProtocol.ts
        /src/native/common/generatedipc/CommonNativeFacadeReceiveDispatcher.ts
        /src/native/common/generatedipc/DesktopFacadeReceiveDispatcher.ts
        /src/native/common/generatedipc/InterWindowEventFacadeReceiveDispatcher.ts
        /src/native/common/generatedipc/MobileFacadeReceiveDispatcher.ts
        /src/native/common/generatedipc/WebGlobalDispatcher.ts
        /src/native/common/generatedipc/CommonSystemFacadeSendDispatcher.ts
        /src/native/common/generatedipc/MobileSystemFacadeSendDispatcher.ts
        /src/native/common/generatedipc/ThemeFacadeSendDispatcher.ts
        /src/native/common/generatedipc/SearchTextInAppFacadeSendDispatcher.ts
        /src/native/common/generatedipc/SettingsFacadeSendDispatcher.ts
        /src/native/common/generatedipc/DesktopSystemFacadeSendDispatcher.ts
native-main-6dnfxQBS.js  15.865234375K
        /src/native/main/UpdatePrompt.ts
        /src/native/main/DesktopNativeTransport.ts
        /src/native/main/WebauthnNativeBridge.ts
        /src/native/main/WebDesktopFacade.ts
        /src/native/main/WebMobileFacade.ts
        /src/native/main/WebCommonNativeFacade.ts
        /src/native/main/WebInterWindowEventFacade.ts
        /src/native/main/IosNativeTransport.ts
        /src/native/main/AndroidNativeTransport.ts
        /src/native/main/NativeInterfaceMain.ts
        /src/native/main/NativePushServiceApp.ts
        /src/native/main/NativeInterfaceFactory.ts
        /src/gui/SearchInPageOverlay.ts
        /src/native/main/OpenMailboxHandler.ts
polyfill-helpers-VB3Z8Cpa.js  0.5966796875K
        commonjsHelpers.js
        commonjs-dynamic-modules
sanitizer-XaDr8x_M.js  26.158203125K
        libs/purify.js?commonjs-module
        libs/purify.js
        /src/misc/HtmlSanitizer.ts
search-Ea0aa8WK.js  43.9052734375K
        /src/search/view/SearchListView.ts
        /src/search/SearchBarOverlay.ts
        /src/search/view/SearchRouter.ts
        /src/search/SearchBar.ts
        /src/search/view/SearchViewModel.ts
        /src/search/view/SearchView.ts
main-yvWL6ck5.js  225.0986328125K
        /src/api/main/WorkerClient.ts
        /src/api/main/EventController.ts
        /src/api/main/EntropyCollector.ts
        /src/search/model/SearchModel.ts
        /src/misc/FormatValidator.ts
        /src/misc/Formatter.ts
        /src/misc/parsing/MailAddressParser.ts
        /src/mail/model/MailUtils.ts
        /src/api/main/UserError.ts
        /src/mail/model/MailModel.ts
        /src/gui/Notifications.ts
        /src/api/main/ProgressTracker.ts
        /src/mail/model/MinimizedMailEditorViewModel.ts
        /src/misc/credentials/CredentialsProvider.ts
        /src/misc/credentials/CredentialsKeyProvider.ts
        /src/misc/credentials/NativeCredentialsEncryption.ts
        /src/misc/credentials/CredentialsKeyMigrator.ts
        /src/misc/credentials/CredentialsProviderFactory.ts
        /src/gui/CompletenessIndicator.ts
        /src/gui/dialogs/ProgressDialog.ts
        /src/file/FileController.ts
        /src/misc/2fa/SecondFactorAuthView.ts
        /src/misc/2fa/SecondFactorUtils.ts
        /src/misc/2fa/SecondFactorAuthDialog.ts
        /src/misc/2fa/SecondFactorHandler.ts
        /src/misc/2fa/webauthn/WebauthnClient.ts
        /src/misc/2fa/webauthn/WebauthnTypes.ts
        /src/misc/2fa/webauthn/BrowserWebauthn.ts
        /packages/tutanota-usagetests/dist/model/Stage.js
        /packages/tutanota-usagetests/dist/model/UsageTest.js
        /packages/tutanota-usagetests/dist/storage/PingAdapter.js
        /packages/tutanota-usagetests/dist/model/UsageTestController.js
        /packages/tutanota-usagetests/dist/index.js
        /src/misc/UsageTestModel.ts
        /src/api/main/PageContextLoginListener.ts
        /src/file/FileControllerBrowser.ts
        /src/file/FileControllerNative.ts
        /src/misc/news/NewsModel.ts
        /src/misc/WebsocketConnectivityModel.ts
        /src/api/main/OperationProgressTracker.ts
        /src/gui/InfoMessageHandler.ts
        /src/mail/model/InboxRuleHandler.ts
        /src/gui/ScopedRouter.ts
        /src/contacts/model/ContactUtils.ts
        /src/contacts/model/NativeContactsSyncManager.ts
        /src/api/main/MainLocator.ts
        /src/gui/dialogs/ShortcutDialog.ts
        /src/misc/SubscriptionDialogs.ts
        /src/misc/ClipboardUtils.ts
        /src/misc/ErrorReporter.ts
        /src/misc/ErrorHandlerImpl.ts
        /src/gui/main-styles.ts
        /src/misc/NavShortcuts.ts
        /src/misc/LoginUtils.ts
        /src/misc/OutOfOfficeNotificationUtils.ts
        /src/gui/nav/ViewSlider.ts
        /src/gui/Header.ts
        /src/gui/ListColumnWrapper.ts
        /src/gui/SelectableRowContainer.ts
        /src/gui/nav/NavFunctions.ts
        /src/misc/news/NewsList.ts
        /src/misc/news/NewsDialog.ts
        /src/gui/nav/DrawerMenu.ts
        /src/gui/FolderColumnView.ts
        /src/gui/SidebarSection.ts
        /src/gui/nav/BottomNav.ts
        /src/gui/BaseTopLevelView.ts
        /src/gui/cards.ts
        /src/gui/PinchZoom.ts
        /src/gui/ScrollSelectList.ts
        /src/gui/SearchDropDown.ts
        /src/gui/MailRecipientsTextField.ts
        /src/gui/AttachmentBubble.ts
        /src/misc/passwords/PasswordUtils.ts
        /src/gui/BackgroundColumnLayout.ts
        /src/gui/DesktopToolbars.ts
        /src/gui/SelectAllCheckbox.ts
        /src/gui/MobileBottomActionBar.ts
        /src/gui/BaseMobileHeader.ts
        /src/gui/MobileHeader.ts
        /src/misc/LazySearchBar.ts
        /src/gui/MultiselectMobileHeader.ts
        /src/gui/EnterMultiselectIconButton.ts
        /src/misc/ListModel.ts
        /src/gui/LoginScreenHeader.ts
        /src/misc/news/MoreInfoLink.ts
        /src/gui/dialogs/SelectCredentialsEncryptionModeDialog.ts
        /src/misc/passwords/PasswordGenerator.ts
        /src/misc/passwords/PasswordGeneratorDialog.ts
        /src/api/main/UpgradeRequiredError.ts
        /src/gui/dialogs/SpellcheckLanguageDialog.ts
        /src/misc/MailboxPropertiesUtils.ts
        /src/misc/Website.ts
        /src/api/main/RecipientsModel.ts
        /src/misc/news/items/ReferralLinkViewer.ts
        /src/search/model/SearchUtils.ts
        /src/misc/SanitizedTextViewModel.ts
        /src/misc/SimpleTextViewModel.ts
        /src/serviceworker/ServiceWorkerClient.ts
        /src/api/main/UserController.ts
        /src/misc/RecipientsSearchModel.ts
        /src/misc/news/items/UsageOptInNews.ts
        /src/misc/news/items/RecoveryCodeNews.ts
        /src/misc/news/items/PinBiometricsNews.ts
        /src/misc/news/items/ReferralLinkNews.ts
        /src/misc/news/items/NewPlansNews.ts
        /src/misc/news/items/NewPlansOfferEndingNews.ts
        /src/contacts/model/ContactModel.ts
sharing-6rxKJWJ8.js  17.853515625K
        /src/sharing/GroupUtils.ts
        /src/sharing/GroupSharingUtils.ts
        /src/sharing/model/GroupSharingModel.ts
        /src/sharing/GroupGuiUtils.ts
        /src/sharing/view/GroupSharingDialog.ts
        /src/sharing/model/ReceivedGroupInvitationsModel.ts
        /src/sharing/view/ReceivedGroupInvitationDialog.ts
        /src/sharing/view/GroupInvitationFolderRow.ts
settings-p3FVcxJY.js  357.4248046875K
        /src/termination/TerminationViewModel.ts
        /src/subscription/CancellationReasonInput.ts
        /src/termination/TerminationForm.ts
        /src/termination/TerminationView.ts
        /src/settings/SettingsFolder.ts
        /src/settings/login/RecoverCodeDialog.ts
        libs/qrcode.js?commonjs-module
        libs/qrcode.js
        /src/settings/login/secondfactor/SecondFactorEditModel.ts
        /src/settings/login/secondfactor/SecondFactorEditDialog.ts
        /src/settings/login/secondfactor/SecondFactorsEditForm.ts
        /src/settings/PasswordForm.ts
        /src/settings/login/ChangePasswordDialogs.ts
        /src/settings/login/LoginSettingsViewer.ts
        /src/settings/AddSpamRuleDialog.ts
        /src/settings/LoadingUtils.ts
        /src/settings/DomainDnsStatus.ts
        /src/settings/emaildomain/DnsRecordTable.ts
        /src/settings/emaildomain/VerifyDnsRecordsPage.ts
        /src/settings/CheckDomainDnsStatusDialog.ts
        /src/settings/SettingsExpander.ts
        /src/settings/ExpandableTable.ts
        /src/settings/RejectedSendersInfoDialog.ts
        /src/settings/SelectMailAddressForm.ts
        /src/settings/mailaddress/MailAddressesUtils.ts
        /src/subscription/SubscriptionUtils.ts
        /src/settings/mailaddress/MailAddressTableModel.ts
        /src/settings/mailaddress/AddAliasDialog.ts
        /src/settings/mailaddress/MailAddressTable.ts
        /src/settings/emaildomain/AddEmailAddressesPage.ts
        /src/settings/emaildomain/VerifyOwnershipPage.ts
        /src/settings/emaildomain/EnterDomainPage.ts
        /src/settings/emaildomain/AddDomainWizard.ts
        /src/subscription/DeleteAccountDialog.ts
        /src/settings/GlobalSettingsViewer.ts
        /src/settings/DesktopUpdateHelpLabel.ts
        /src/settings/DesktopSettingsViewer.ts
        /src/settings/AddInboxRuleDialog.ts
        /src/settings/IdentifierListViewer.ts
        /src/settings/EditOutOfOfficeNotificationDialogModel.ts
        /src/settings/EditOutOfOfficeNotificationDialog.ts
        /src/settings/EditSignatureDialog.ts
        /src/settings/OfflineStorageSettings.ts
        /src/settings/MailSettingsViewer.ts
        /src/subscription/PriceUtils.ts
        /src/subscription/BuyDialog.ts
        /src/settings/groups/GroupDetailsView.ts
        /src/settings/ImportUsersViewer.ts
        /src/settings/UserViewer.ts
        /src/subscription/BuyOptionBox.ts
        /src/subscription/FeatureListProvider.ts
        /src/subscription/SubscriptionSelector.ts
        /src/subscription/UpgradeSubscriptionPage.ts
        /src/subscription/InvoiceDataInput.ts
        /src/subscription/SimplifiedCreditCardInput.ts
        /src/subscription/SimplifiedCreditCardInputModel.ts
        /src/subscription/PaymentMethodInput.ts
        /src/subscription/InvoiceAndPaymentDataPage.ts
        /src/subscription/UpgradeCongratulationsPage.ts
        /src/subscription/TermsAndConditions.ts
        /src/subscription/Captcha.ts
        /src/subscription/SignupForm.ts
        /src/subscription/SignupPage.ts
        /src/subscription/UpgradeConfirmSubscriptionPage.ts
        /src/subscription/UpgradeSubscriptionWizard.ts
        /src/settings/AddUserDialog.ts
        /src/settings/UserListView.ts
        /src/settings/groups/AddGroupDialog.ts
        /src/settings/LocalAdminGroupInfoModel.ts
        /src/settings/groups/GroupDetailsModel.ts
        /src/settings/groups/GroupListView.ts
        /src/settings/EditNotificationEmailDialog.ts
        /src/settings/SetDomainCertificateDialog.ts
        /src/settings/whitelabel/WhitelabelBrandingDomainSettings.ts
        /src/settings/whitelabel/CustomColorEditorPreview.ts
        /src/settings/whitelabel/CustomColorEditor.ts
        /src/settings/whitelabel/EditCustomColorsDialog.ts
        /src/settings/whitelabel/CustomColorsEditorViewModel.ts
        /src/settings/whitelabel/WhitelabelThemeSettings.ts
        /src/settings/whitelabel/WhitelabelImprintAndPrivacySettings.ts
        /src/settings/whitelabel/WhitelabelRegistrationSettings.ts
        /src/settings/whitelabel/WhitelabelCustomMetaTagsSettings.ts
        /src/settings/whitelabel/WhitelabelStatusSettings.ts
        /src/settings/whitelabel/WhitelabelNotificationEmailSettings.ts
        /src/settings/whitelabel/WhitelabelGermanLanguageFileSettings.ts
        /src/settings/whitelabel/WhitelabelSettingsViewer.ts
        /src/subscription/SwitchSubscriptionDialogModel.ts
        /src/subscription/SwitchToBusinessInvoiceDataDialog.ts
        /src/subscription/SwitchSubscriptionDialog.ts
        /src/subscription/SignOrderProcessingAgreementDialog.ts
        /src/subscription/giftcards/GiftCardUtils.ts
        /src/subscription/giftcards/GiftCardMessageEditorField.ts
        /src/subscription/giftcards/PurchaseGiftCardDialog.ts
        /src/subscription/SubscriptionViewer.ts
        /src/subscription/InvoiceDataDialog.ts
        /src/subscription/PaymentDataDialog.ts
        /src/subscription/PaymentViewer.ts
        /src/settings/AppearanceSettingsViewer.ts
        /src/settings/KnowledgeBaseEditorModel.ts
        /src/settings/KnowledgeBaseEditor.ts
        /src/settings/KnowledgeBaseListView.ts
        /src/settings/TemplateEditorModel.ts
        /src/settings/TemplateEditor.ts
        /src/settings/DummyTemplateListView.ts
        /src/settings/SettingsFolderRow.ts
        /src/settings/UserDataExporter.ts
        /src/settings/ReferralSettingsViewer.ts
        /src/settings/AboutDialog.ts
        /src/settings/TemplateDetailsViewer.ts
        /src/settings/TemplateListView.ts
        /src/settings/ContactsSettingsViewer.ts
        /src/settings/SettingsView.ts
        /src/settings/mailaddress/OwnMailAddressNameChanger.ts
        /src/settings/mailaddress/AnotherUserMailAddressNameChanger.ts
        /src/subscription/giftcards/RedeemGiftCardWizard.ts
ui-extra-vPLmKkN-.js  6.904296875K
        /src/login/recover/RecoverLoginDialog.ts
        /src/support/FaqModel.ts
        /src/support/SupportDialog.ts
        /src/login/recover/TakeOverDeletedAddressDialog.ts
worker-lazy-ELbjqHW0.js  53.20703125K
        /src/api/worker/facades/lazy/BookingFacade.ts
        /src/api/worker/facades/lazy/CounterFacade.ts
        /src/api/worker/facades/lazy/GroupManagementFacade.ts
        /src/api/worker/facades/lazy/UserManagementFacade.ts
        /src/api/worker/facades/lazy/CustomerFacade.ts
        /src/api/worker/facades/lazy/BlobFacade.ts
        /src/api/worker/facades/lazy/MailFacade.ts
        /src/api/worker/facades/lazy/CalendarFacade.ts
        /src/api/worker/facades/lazy/MailAddressFacade.ts
        /src/api/worker/facades/lazy/ShareFacade.ts
        /src/api/worker/facades/lazy/GiftCardFacade.ts
        /src/api/worker/facades/lazy/ConfigurationDatabase.ts
worker-search-ovLYaCwp.js  54.0634765625K
        /src/api/worker/search/DbFacade.ts
        /src/api/worker/search/SearchIndexEncoding.ts
        /src/api/worker/search/IndexUtils.ts
        /src/api/worker/search/IndexTables.ts
        /src/api/worker/search/ContactIndexer.ts
        /src/api/worker/search/MailIndexer.ts
        /src/api/worker/search/IndexerCore.ts
        /src/api/worker/search/SuggestionFacade.ts
        /src/api/worker/search/Indexer.ts
        /src/api/worker/search/SearchFacade.ts
If you do not supply "output.name", you may not be able to access the exports of an IIFE bundle.
prepare mobile build for build type dist
unlinking  build/wasm
unlinking  build/images/custom-md-trash-empty.svg
unlinking  build/images/custom-reply-all.svg
unlinking  build/images/logo-favicon-152.png
unlinking  build/images/logo-favicon-192.png
unlinking  build/images/logo-favicon.svg
unlinking  build/images/logo-red-solo-mactemplate.afphoto
unlinking  build/images/logo-red.svg
unlinking  build/images/logo-solo-red-macos.svg
unlinking  build/images/logo-solo-red.svg
unlinking  build/app.js.map
unlinking  build/calendar-view-vM0-Ai4P.js.map
unlinking  build/common-4v31ka00.js.map
unlinking  build/common-min-g4mXDxm1.js.map
unlinking  build/contacts-t9t14rkB.js.map
unlinking  build/date-brtlBV80.js.map
unlinking  build/date-gui-Gzrcvsyx.js.map
unlinking  build/gui-base-_jExEZnA.js.map
unlinking  build/invoice-wYjpwPd7.js.map
unlinking  build/jszip-TGEAmg11.js.map
unlinking  build/linkify-h7icspiz.js.map
unlinking  build/login-L0dBcvPO.js.map
unlinking  build/mail-editor-h_g6drwz.js.map
unlinking  build/mail-view-PGDZ55zi.js.map
unlinking  build/main-yvWL6ck5.js.map
unlinking  build/native-common-Pv5M4cD9.js.map
unlinking  build/native-main-6dnfxQBS.js.map
unlinking  build/polyfill-helpers-VB3Z8Cpa.js.map
unlinking  build/sanitizer-XaDr8x_M.js.map
unlinking  build/search-Ea0aa8WK.js.map
unlinking  build/settings-p3FVcxJY.js.map
unlinking  build/sharing-6rxKJWJ8.js.map
unlinking  build/sw.js.map
unlinking  build/translation-ar-h-vlWZn4.js.map
unlinking  build/translation-be-PzSP66I7.js.map
unlinking  build/translation-bg-TUp-v0Ld.js.map
unlinking  build/translation-ca-9-OXbWad.js.map
unlinking  build/translation-cs-IEq46GFO.js.map
unlinking  build/translation-da-bMvxFbxX.js.map
unlinking  build/translation-de_sie-33xMSFJO.js.map
unlinking  build/translation-de-UTajvy0t.js.map
unlinking  build/translation-el-qEDx2rk0.js.map
unlinking  build/translation-en-7YDPeRTQ.js.map
unlinking  build/translation-es-O_J20r8K.js.map
unlinking  build/translation-et-VciOJWJM.js.map
unlinking  build/translation-fa_ir-0l90TdbE.js.map
unlinking  build/translation-fi-vBr7Uizy.js.map
unlinking  build/translation-fr-D7S3BHEj.js.map
unlinking  build/translation-gl-VX-H6wit.js.map
unlinking  build/translation-he-Ibnms5RG.js.map
unlinking  build/translation-hi-HlYqHIKU.js.map
unlinking  build/translation-hr-NdNxNWcV.js.map
unlinking  build/translation-hu-kyPnf71u.js.map
unlinking  build/translation-id-GalJeukq.js.map
unlinking  build/translation-it-NOZlnNzl.js.map
unlinking  build/translation-ja-nrCS2-EN.js.map
unlinking  build/translation-ko-c1kz5AeT.js.map
unlinking  build/translation-lt-b2QDobFN.js.map
unlinking  build/translation-lv-lez81bMx.js.map
unlinking  build/translation-nl-zTZcU8v4.js.map
unlinking  build/translation-no-kdMkIgiz.js.map
unlinking  build/translation-pl-sul8YuRZ.js.map
unlinking  build/translation-pt_br-Im73HeXP.js.map
unlinking  build/translation-pt_pt-EgfIR8Mc.js.map
unlinking  build/translation-ro-FKk6fO17.js.map
unlinking  build/translation-ru-_kC12cBg.js.map
unlinking  build/translation-si-_usvqDsn.js.map
unlinking  build/translation-sk-chRSLP4I.js.map
unlinking  build/translation-sl-AKb7-94A.js.map
unlinking  build/translation-sr_cyrl-z7waJmeX.js.map
unlinking  build/translation-sv-OqMQ15aA.js.map
unlinking  build/translation-tr-aybTH5PU.js.map
unlinking  build/translation-uk-WkhWa4Gv.js.map
unlinking  build/translation-vi-_BCOIa8C.js.map
unlinking  build/translation-zh_hant-o7aaLYMd.js.map
unlinking  build/translation-zh-fMwg6SRC.js.map
unlinking  build/ui-extra-vPLmKkN-.js.map
unlinking  build/worker-lazy-ELbjqHW0.js.map
unlinking  build/worker-search-ovLYaCwp.js.map
unlinking  build/worker.js.map
rm  build/index.html
$ rm -r build/app-android
rm: cannot remove 'build/app-android': No such file or directory
$ cd ./app-android
$ ./gradlew assembleTutaorelease
Downloading https://services.gradle.org/distributions/gradle-7.5-bin.zip
...........10%............20%...........30%............40%...........50%............60%...........70%............80%...........90%............100%

Welcome to Gradle 7.5!

Here are the highlights of this release:
 - Support for Java 18
 - Support for building with Groovy 4
 - Much more responsive continuous builds
 - Improved diagnostics for dependency resolution

For more details see https://docs.gradle.org/7.5/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings generic class cache for settings file '/workspaces/tutanota/app-android/settings.gradle' (/home/codespace/.gradle/caches/7.5/scripts/ek3xnpoxvsp1p3cgdy2egar9t).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 65

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 12s
file:///workspaces/tutanota/node_modules/zx/build/core.js:140
            let output = new ProcessOutput(code, signal, stdout, stderr, combined, message);
                         ^

ProcessOutput [Error]: 
FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings generic class cache for settings file '/workspaces/tutanota/app-android/settings.gradle' (/home/codespace/.gradle/caches/7.5/scripts/ek3xnpoxvsp1p3cgdy2egar9t).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 65

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 12s
    at buildAndroid (file:///workspaces/tutanota/android.js:91:9)
    exit code: 1
    at ChildProcess.<anonymous> (file:///workspaces/tutanota/node_modules/zx/build/core.js:140:26)
    at ChildProcess.emit (node:events:518:28)
    at maybeClose (node:internal/child_process:1105:16)
    at ChildProcess._handle.onexit (node:internal/child_process:305:5)
    at Process.callbackTrampoline (node:internal/async_hooks:130:17) {
  _code: 1,
  _signal: null,
  _stdout: 'Downloading https://services.gradle.org/distributions/gradle-7.5-bin.zip\n' +
    '...........10%............20%...........30%............40%...........50%............60%...........70%............80%...........90%............100%\n' +
    '\n' +
    'Welcome to Gradle 7.5!\n' +
    '\n' +
    'Here are the highlights of this release:\n' +
    ' - Support for Java 18\n' +
    ' - Support for building with Groovy 4\n' +
    ' - Much more responsive continuous builds\n' +
    ' - Improved diagnostics for dependency resolution\n' +
    '\n' +
    'For more details see https://docs.gradle.org/7.5/release-notes.html\n' +
    '\n' +
    'Starting a Gradle Daemon (subsequent builds will be faster)\n',
  _stderr: '\n' +
    'FAILURE: Build failed with an exception.\n' +
    '\n' +
    '* What went wrong:\n' +
    "Could not open settings generic class cache for settings file '/workspaces/tutanota/app-android/settings.gradle' (/home/codespace/.gradle/caches/7.5/scripts/ek3xnpoxvsp1p3cgdy2egar9t).\n" +
    "> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 65\n" +
    '\n' +
    '* Try:\n' +
    '> Run with --stacktrace option to get the stack trace.\n' +
    '> Run with --info or --debug option to get more log output.\n' +
    '> Run with --scan to get full insights.\n' +
    '\n' +
    '* Get more help at https://help.gradle.org\n' +
    '\n' +
    'BUILD FAILED in 12s\n',
  _combined: 'Downloading https://services.gradle.org/distributions/gradle-7.5-bin.zip\n' +
    '...........10%............20%...........30%............40%...........50%............60%...........70%............80%...........90%............100%\n' +
    '\n' +
    'Welcome to Gradle 7.5!\n' +
    '\n' +
    'Here are the highlights of this release:\n' +
    ' - Support for Java 18\n' +
    ' - Support for building with Groovy 4\n' +
    ' - Much more responsive continuous builds\n' +
    ' - Improved diagnostics for dependency resolution\n' +
    '\n' +
    'For more details see https://docs.gradle.org/7.5/release-notes.html\n' +
    '\n' +
    'Starting a Gradle Daemon (subsequent builds will be faster)\n' +
    '\n' +
    'FAILURE: Build failed with an exception.\n' +
    '\n' +
    '* What went wrong:\n' +
    "Could not open settings generic class cache for settings file '/workspaces/tutanota/app-android/settings.gradle' (/home/codespace/.gradle/caches/7.5/scripts/ek3xnpoxvsp1p3cgdy2egar9t).\n" +
    "> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 65\n" +
    '\n' +
    '* Try:\n' +
    '> Run with --stacktrace option to get the stack trace.\n' +
    '> Run with --info or --debug option to get more log output.\n' +
    '> Run with --scan to get full insights.\n' +
    '\n' +
    '* Get more help at https://help.gradle.org\n' +
    '\n' +
    'BUILD FAILED in 12s\n'
}

Node.js v20.11.0
@Osiris-Team ➜ /workspaces/tutanota (master) $ 
```
</details>

**Expected behavior**
Complete errorless build.

**Screenshots**
-


**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Version [e.g. 22]

**Additional context**
Building in in github codespaces.
Followed the steps in the readme, to build the androidn app, without generating certificates, since these should only be needed at release.

--- a/app-android/gradle/wrapper/gradle-wrapper.properties
+++ b/app-android/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
 #Thu Feb 01 10:01:17 CET 2024
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
 distributionPath=wrapper/dists
 zipStorePath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME

